### ISSUE TITLE
Not able to use custom endpoint to pull memory stats

### ISSUE THREAD

**Issue by luyanqi** [F5Networks/f5-telemetry-streaming#134]
### Environment
 * Telemetry Streaming Version: 1.20.0
 * BIG-IP Version:  15.1.2.1.0.9.10

### Summary
Use custom Endpoint to get Memory utilization, TS declaration successfully pushed to BIG-IP
When getting 500 error when query the pull consumer URL

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "Telemetry",
    "controls": {
     "class": "Controls",
     "logLevel": "debug"
               },
    "Endpoints_MemoryUtilization": {
        "class": "Telemetry_Endpoints",
        "items": {
            "MemoryUtilization": {
                "name": "MemoryUtilization",
                "path": "/mgmt/tm/sys/memory/stats"
            }
        }
    },
    "Custom_System": {
        "class": "Telemetry_System",
        "systemPoller": [
            "Custom_System_Poller1",
            {
                "interval": 0
            }
        ]
    },
    "Custom_System_Poller1": {
        "class": "Telemetry_System_Poller",
        "interval": 0,
        "endpointList": [
            "Endpoints_MemoryUtilization/MemoryUtilization",
            {
                "path": "/mgmt/tm/sys/memory/stats",
                "name": "MemoryUtilization"
            }
        ]
    },
    "My_Listener": {
        "class": "Telemetry_Listener",
        "port": 6514
    },
    "metrics": {
        "class": "Telemetry_Pull_Consumer",
        "type": "Prometheus",
        "trace": "/var/tmp/telemetry_kafka_trace1.log",
        "systemPoller": ["Custom_System_Poller1"]
    }
}
```
2. Observe the following error response:
when query Pull comsumer URL /mgmt/shared/telemetry/pullconsumer/metrics, receive 500 Internal Server Error
The following error observed in restnoded.log. The same config works for TS 1.18
```
Error: A metric with the name f5_MemoryUtilization_memory_subsys_stats_memory_subsys_html_rule_stats_allocated has already been registered.
    at Registry.registerMetric (/var/config/rest/iapps/f5-telemetry/node_modules/prom-client/lib/registry.js:71:10)
    at Gauge.config.registers.forEach.registryInstance (/var/config/rest/iapps/f5-telemetry/node_modules/prom-client/lib/gauge.js:72:21)
    at Array.forEach (<anonymous>)
    at new Gauge (/var/config/rest/iapps/f5-telemetry/node_modules/prom-client/lib/gauge.js:71:20)
    at Object.keys.forEach (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers/Prometheus/index.js:137:23)
    at Array.forEach (<anonymous>)
    at applyPrometheusFormatting (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers/Prometheus/index.js:134:23)
    at Promise (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers/Prometheus/index.js:86:9)
    at Promise._execute (/usr/share/rest/node/node_modules/bluebird/js/release/debuggability.js:303:9)
    at Promise._resolveFromExecutor (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:79:10)
    at Object.module.exports [as consumer] (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers/Prometheus/index.js:74:12)
    at invokeConsumer (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers.js:75:21)
    at Promise.resolve.then.then.pollerData (/var/config/rest/iapps/f5-telemetry/lib/pullConsumers.js:53:29)
    at tryCatcher (/usr/share/rest/node/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:638:18)
    at Promise._resolveCallback (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:454:14)
    at Promise._settlePromiseFromHandler (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:524:17)
Fri, 28 May 2021 02:12:20 GMT - info: [telemetry] 500 GET /shared/telemetry/pullconsumer/metrics
```
### Expected Behavior
Able to query /mgmt/shared/telemetry/pullconsumer/metrics and get Memory Utiliation data

### Actual Behavior
500 internal error

**Comment by enceladus-beep** [comment]
internal SR 1-7441345612

**Comment by dstokesf5** [comment]
Confirmed the bug. I have added it to our internal product backlog as AUTOTOOL-2543. Thank you for your feedback.

**Comment by dstokesf5** [comment]
This issue has been resolved with the 1.22.0 release of Telemetry Streaming.
